content/docs/user-guide/experiment-management/experiments-overview.md

```diff
@@ -0,0 +1,71 @@
+# DVC Experiments Overview
+
+DVC Experiments are captured automatically by DVC when [run]. Each experiment
+creates and tracks a variation of your data science project based on the changes
+in your <abbr>workspace</abbr>.
+
+Experiments preserve a connection to the latest commit in the current branch
+(Git `HEAD`) as their parent or _baseline_, but do not form part of the regular
+Git tree or workflow (unless you make them [persistent]). This prevents
+polluting Git namespaces and bloating the repo unnecessarily.
```

Comment:
Another option might be to reference that it's explained in more detail below.

Reply:
> this paragraph feels a little abstract

Updated a little (3af2f9ac)

> Do we want to avoid terms like ephemeral/lightweight branches

Yes, those terms would also require definition and they're an implementation detail anyway. We're trying to stick to what matters most from the user perspective.

> reference that it's explained in more detail below

The title of the hidden section which is right after this paragraph should already serve as a reference or call-out for more info, I think.